AdyenActions/Components/Await/AwaitComponent.swift

```diff
@@ -22,35 +22,53 @@ public final class AwaitComponent: ActionComponent, Cancellable {
     /// :nodoc:
     public let requiresModalPresentation: Bool = true
     
-    /// The Component UI style.
-    public let style: AwaitComponentStyle
+    /// The await component configurations.
+    public struct Configuration {
+        
+        /// The component UI style.
+        public var style: AwaitComponentStyle = .init()
```

Comment:
do we need to set the initial value here if it is set in the `init`?

Reply:
it doesn't hurt, does it?

Reply:
it's just settings a value which will immediately be overwritten in `init`. if it had no initial value - compiler would check if every version of `init` actually sets a value for this field.

Reply:
ok will fix it with the other one in another PR